crates/task-impls/src/network.rs

```diff
@@ -652,6 +652,13 @@ impl<
                     TransmitType::Direct(to),
                 ))
             }
+            HotShotEvent::HighQcSend(quorum_cert, leader, sender) => Some((
+                sender,
+                MessageKind::Consensus(SequencingMessage::General(
+                    GeneralConsensusMessage::HighQc(quorum_cert.to_qc()),
```

Comment:
can we change `GeneralConsensusMessage::HighQc` to a `QuorumCertificate2`?

it's not deployed so I think we can make the breaking change, and I think we would be gated on the same version (epochs)